src/java/htsjdk/samtools/SAMRecord.java

```diff
@@ -505,6 +505,38 @@ public int getReferencePositionAtReadPosition(final int offset) {
     }
 
     /**
+     * @param rec record to use
+     * @param pos 1-based reference position
+     * @return 1-based inclusive offset position into the unclipped sequence at a given reference position,
```

Comment:
I wanted this function to be the "inverse" of getReferencePositionAtReadPosition which assumes 1-based read position. 

I plan to keep the 1-based read-pos unless I hear screaming (literally :-) )